docs/examples/AccelerometerExample/src/AccelerometerExample.ls

```diff
@@ -59,10 +68,13 @@ package
             sprite.y = Math.clamp(sprite.y, 0, stage.stageHeight);
         }
 
-        protected function onAccelerometerData(x:Number, y:Number, z:Number) 
+        protected function onAccelerometerData(type:MobileSensorType, x:Number, y:Number, z:Number)
```

Comment:
This function could get called for other sensor triples as well if you enabled them and that's why you need the check, right? (If you had other sensors enabled this function would probably be more appropriately named ´onSensorTriple´?)